src/pages/index.js

```diff
@@ -54,7 +54,7 @@ function IndexPage() {
         </h2>
       </section>
 
-      <section className="main-section">
+      <section className="main-section section-content-2">
```

Comment:
Is this the same use case as what I found here below?

[className](https://www.w3schools.com/jsref/prop_html_classname.asp)

<img width="1003" alt="Screen Shot 2021-05-10 at 5 37 30 PM" src="https://user-images.githubusercontent.com/66753884/117732983-69ac5d00-b1b6-11eb-891a-2420b4ba68b8.png">